### ISSUE TITLE
Multiple docx conversion with pandoc reuses wrong media files

### ISSUE THREAD

**Issue by SnarkyWraith** [snsinahub-org/docupage-multiple#19]
Try doing a bulk upload/pandoc conversion of two .docx files that both have different embedded images.  I think because word docs have their own internal "file system" for their embedded media (that's where the media directory comes from on the export and they are all sequentially named image1, image2, etc), when you end up reusing the same media directory, pandoc is either overwriting the existing images with the same name or assuming the image it needs already exists and is using that (haven't figured out which yet).

Bottom line, I tried to convert two word docs with images and then viewed the resulting markdown.  One of them was re-using the other doc's images.  Think the solution is to have unique media extraction directories for each doc.
